### PR TITLE
added description of LinksHub

### PR DESCRIPTION
**Pull request by CBID2**
## Fixes Issue
Closes[#331](https://github.com/rupali-codes/LinksHub/issues/331)

## Changes proposed

I added the website's description to the hero section. 
## Note to reviewers
I never worked with Typescript before so constructive feedback would be helpful.

### CODE REVIEW

**Comment by ujjawaltyagii** [comment]
I just checked the changes in preview and correct me if I am wrong as I think description is looking a bit odd I mean the placement! 

image

**Review by Deva-1903 — APPROVED** [review]

**Comment by imbeshat** [comment]
> I just checked the changes in preview and correct me if I am wrong as I think description is looking a bit odd I mean the placement!

The placement looks odd to me as well.

If I am not wrong, it should be something like this:

image

**Comment by rupali-codes** [comment]
@CBID2 can we make it same width as logo?

**Comment by CBID2** [comment]
> @CBID2 can we make it same width as logo?

Sure @rupali-codes! I'm going to need your help though. For some reason, I was not able to see the entire text in the file you sent.

**Comment by rupali-codes** [comment]
@CBID2 there you go! 😂😂😂
image

**Comment by ujjawaltyagii** [comment]
It seems that there is still issue with the description text and it's styling, let me know if you need any help.

**Comment by CBID2** [comment]
> It seems that there is still issue with the description text and it's styling, let me know if you need any help.

I could definitely use your suggestions @ujjawaltyagii! 😊

**Comment by rupali-codes** [comment]
> Can someone help me with this?

Hey Chrissy, yeah sure, what issues are you getting?

**Comment by CBID2** [comment]
> > Can someone help me with this?
> 
> Hey Chrissy, yeah sure, what issues are you getting?

Hey sorry for the late reply @rupali-codes. I can't seem to structure the paragraph underneath the logo. Maybe you can add a suggestion and I can commit it?

**Comment by rupali-codes** [comment]
> > > Can someone help me with this?
> > 
> > 
> > Hey Chrissy, yeah sure, what issues are you getting?
> 
> Hey sorry for the late reply @rupali-codes. I can't seem to structure the paragraph underneath the logo. Maybe you can add a suggestion and I can commit it?

Sure

**Comment by TreciaKS** [comment]
> @CBID2 can we make it same width as logo?

@rupali-codes I have achieved this. Let me know if I can give my suggestion :)
Screenshot 2023-04-15 at 14 52 58

**Comment by CBID2** [comment]
> > @CBID2 can we make it same width as logo?
> 
> @rupali-codes I have achieved this. Let me know if I can give my suggestion :)
> 
> The only question is: do you want the description text to be the same font size as the text animation below it?
> 
> Screenshot 2023-04-15 at 14 52 58

I like that @parma1111! 😁 Just add your suggestions to my PR via the “Suggestion Review” section of my [article](https://www.freecodecamp.org/news/code-review-tips/). You'll get green squares that way.

File: pages/index.tsx [hunk 1]
```diff
@@ -24,6 +24,8 @@ export default function Home() {
           <div className="m-auto md:text-7xl text-5xl gap-2 flex items-center justify-center">
             <Logo />
             <span>👾</span>
+              <p> LinksHub aims to provide developers with access to a wide range of free resources and tools that they can use in their work.</p> 
+              <p>These resources include links to free software, libraries, frameworks, and other tools that can be used to build and deploy applications and websites.</p>
           </div>
           <div className=" flex flex-col mt-6 justify-center items-start">
```
Comment (TreciaKS) [inline pages/index.tsx:30]:
```suggestion
          <div className=" flex flex-col mt-6 justify-center items-start w-96 max-md:w-64">
          <p className="text-md"> LinksHub aims to provide developers with access to a wide range of free resources and tools that they can use in their work. These resources include links to free software, libraries, frameworks, and other tools that can be used to build and deploy applications and websites.</p>
          <br/>
```

**Comment by TreciaKS** [comment]
@CBID2 I believe it is best to place the description text within the nexted <div> for the truncated text. In this way, the width (w-96) can be applied on both the description and truncated text and fit the width of the logo. (max-md:w-64) is applied on smaller screens for responsiveness and width consistency. 

Let me know what you think 🤔 ]

**Comment by CBID2** [comment]
> @CBID2 I believe it is best to place the description text within the nexted <div> for the truncated text. In this way, the width (w-96) can be applied on both the description and truncated text and fit the width of the logo. (max-md:w-64) is applied on smaller screens for responsiveness and width consistency. 
> 
> 
> 
> Let me know what you think 🤔 ]

I'm open to it! Add your suggestion

File: pages/index.tsx [hunk 2]
```diff
@@ -24,8 +24,12 @@ export default function Home() {
           <div className="m-auto md:text-7xl text-5xl gap-2 flex items-center justify-center">
             <Logo />
             <span>👾</span>
+              <p> LinksHub aims to provide developers with access to a wide range of free resources and tools that they can use in their work.</p>
```
Comment (TreciaKS) [inline pages/index.tsx:27]:
```suggestion
```

File: pages/index.tsx [hunk 3]
```diff
@@ -24,8 +24,12 @@ export default function Home() {
           <div className="m-auto md:text-7xl text-5xl gap-2 flex items-center justify-center">
             <Logo />
             <span>👾</span>
+              <p> LinksHub aims to provide developers with access to a wide range of free resources and tools that they can use in their work.</p> 
+              <p>These resources include links to free software, libraries, frameworks, and other tools that can be used to build and deploy applications and websites.</p>
```
Comment (TreciaKS) [inline pages/index.tsx:28]:
```suggestion
```

**Comment by parma1111** [comment]
> > @CBID2 I believe it is best to place the description text within the nexted  for the truncated text. In this way, the width (w-96) can be applied on both the description and truncated text and fit the width of the logo. (max-md:w-64) is applied on smaller screens for responsiveness and width consistency.
> > Let me know what you think 🤔 ]
> 
> I'm open to it! Add your suggestion

Now description looks great

File: pages/index.tsx [hunk 4]
```diff
@@ -25,7 +25,9 @@ export default function Home() {
             <Logo />
             <span>👾</span>
           </div>
-          <div className=" flex flex-col mt-6 justify-center items-start">
+          <div className=" flex flex-col mt-6 justify-center items-start w-96 max-md:w-64">
+          <p className="text-md"> LinksHub aims to provide developers with access to a wide range of free resources and tools that they can use in their work. These resources include links to free software, libraries, frameworks, and other tools that can be used to build and deploy applications and websites.</p>
+          <br/>
           <p className="text-xl  ">Navigate through menu for</p>
```
Comment (parma1111) [inline pages/index.tsx:31]:
```suggestion
          <p className="text-xl  ">Navigate through menu for</p>
```

File: pages/index.tsx [hunk 5]
```diff
@@ -25,7 +25,9 @@ export default function Home() {
             <Logo />
             <span>👾</span>
           </div>
-          <div className=" flex flex-col mt-6 justify-center items-start">
+          <div className=" flex flex-col mt-6 justify-center items-start w-96 max-md:w-64">
+          <p className="text-md"> LinksHub aims to provide developers with access to a wide range of free resources and tools that they can use in their work. These resources include links to free software, libraries, frameworks, and other tools that can be used to build and deploy applications and websites.</p>
```
Comment (parma1111) [inline pages/index.tsx:29]:
```suggestion
          <p className="text-md"> LinksHub aims to provide developers with access to a wide range of free resources and tools that they can use in their work. These resources include links to free software, libraries, frameworks, and other tools that can be used to build and deploy applications and websites.</p>
```

File: pages/index.tsx (hunk 4 above)
Comment (rupali-codes) [inline pages/index.tsx:31]:
```suggestion
          <p className="text-md  ">Navigate through menu for</p>
```

File: pages/index.tsx [hunk 6]
```diff
@@ -25,7 +25,9 @@ export default function Home() {
             <Logo />
             <span>👾</span>
           </div>
-          <div className=" flex flex-col mt-6 justify-center items-start">
+          <div className=" flex flex-col mt-6 justify-center items-start w-96 max-md:w-64">
```
Comment (rupali-codes) [inline pages/index.tsx:28]:
```suggestion
          <div className="flex flex-col mt-6 justify-center items-start w-96 max-md:w-64">
```

File: pages/index.tsx [hunk 7]
```diff
@@ -25,7 +25,9 @@ export default function Home() {
...
+          <p className="text-md"> LinksHub aims to provide developers with access to a wide range of free resources and tools that they can use in their work. These resources include links to free software, libraries, frameworks, and other tools that can be used to build and deploy applications and websites.</p>
+          <br/>
           <p className="text-xl  ">Navigate through menu for</p>
           <p className='text-xl text-violet-500'><TypewriterComponent
```
Comment (rupali-codes) [inline pages/index.tsx:32]:
```suggestion
          <p className='text-md text-violet-500'><TypewriterComponent  
```

**Review by rupali-codes — COMMENTED** [review]
Hey Chrissy, just update the text formatting, rest looks great to me

**Comment by CBID2** [comment]
> Hey Chrissy, just update the text formatting, rest looks great to me

Done! :)

**Review by rupali-codes — COMMENTED** [review]
lgtm

**Pull request merged**